src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Regions/Queries/GetRegions/GetRegionsQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿using MediatR;
+
+namespace SFA.DAS.ApprenticeAan.Application.Regions.Queries.GetRegions
```

Comment:
This should be along side the Query Handler.

Reply:
I'm not sure what you mean by this. `GetRegionsQuery` is in the same folder. Does it need to be in the same class?

Reply:
duh moment.